autopilot/contractor/contractor.go

```diff
@@ -243,16 +257,14 @@ func (c *Contractor) refreshContract(ctx *mCtx, contract contract, host api.Host
 	// renew the contract
```

Comment:
A few lines above this one you still return `false` when `ConsensusState` fails.